examples/PlotSpeedTest.py

```diff
@@ -5,53 +5,54 @@
 """
 
 ## Add path to library (just for examples; you do not need this)
-import initExample
+from time import perf_counter
 
+import initExample
 
 from pyqtgraph.Qt import QtGui, QtCore
+
 import numpy as np
 import pyqtgraph as pg
-from pyqtgraph.ptime import time
+
 app = pg.mkQApp("Plot Speed Test")
 
 p = pg.plot()
 p.setWindowTitle('pyqtgraph example: PlotSpeedTest')
-p.setRange(QtCore.QRectF(0, -10, 5000, 20)) 
+p.setRange(QtCore.QRectF(0, -10, 5000, 20))
 p.setLabel('bottom', 'Index', units='B')
 curve = p.plot()
 
-#curve.setFillBrush((0, 0, 100, 100))
-#curve.setFillLevel(0)
+data = np.random.normal(size=(50, 5000))
+ptr = 0
+elapsed = 0
 
-#lr = pg.LinearRegionItem([100, 4900])
-#p.addItem(lr)
 
-data = np.random.normal(size=(50,5000))
-ptr = 0
-lastTime = time()
-fps = None
 def update():
-    global curve, data, ptr, p, lastTime, fps
-    curve.setData(data[ptr%10])
+    global curve, data, ptr, elapsed, ptr
+
     ptr += 1
-    now = time()
-    dt = now - lastTime
-    lastTime = now
-    if fps is None:
-        fps = 1.0/dt
-    else:
-        s = np.clip(dt*3., 0, 1)
-        fps = fps * (1-s) + (1.0/dt) * s
```

Comment:
Any reason for taking out the moving average? Suppressing older samples allows the frame rate estimate be more responsive when changing plot options.

Reply:
Incidentally, I find a measurable difference when replacing `s = np.clip(dt*3., 0, 1)` with `s = min(1, max(0, dt * 3.))`

Reply:
I will change back.